src/app/core/admin/admin-entity/admin-entity-edit/admin-entity-edit.component.ts

```diff
@@ -0,0 +1,86 @@
+import { Component, Input, OnInit, Output, EventEmitter } from "@angular/core";
+import { Entity, EntityConstructor } from "../../../entity/model/entity";
+import { MatButtonModule } from "@angular/material/button";
+import { DialogCloseComponent } from "../../../common-components/dialog-close/dialog-close.component";
+import { MatInputModule } from "@angular/material/input";
+import { ErrorHintComponent } from "../../../common-components/error-hint/error-hint.component";
+import {
+  FormBuilder,
+  FormGroup,
+  FormsModule,
+  ReactiveFormsModule,
+  Validators,
+} from "@angular/forms";
+import { NgIf } from "@angular/common";
+import { MatTabsModule } from "@angular/material/tabs";
+import { MatSlideToggleModule } from "@angular/material/slide-toggle";
+import { FontAwesomeModule } from "@fortawesome/angular-fontawesome";
+import { MatTooltipModule } from "@angular/material/tooltip";
+import { BasicAutocompleteComponent } from "../../../common-components/basic-autocomplete/basic-autocomplete.component";
+
+@Component({
+  selector: "app-admin-entity-edit",
+  standalone: true,
+  templateUrl: "./admin-entity-edit.component.html",
+  styleUrl: "./admin-entity-edit.component.scss",
+  imports: [
+    MatButtonModule,
+    DialogCloseComponent,
+    MatInputModule,
+    ErrorHintComponent,
+    FormsModule,
+    NgIf,
+    MatTabsModule,
+    MatSlideToggleModule,
+    ReactiveFormsModule,
+    FontAwesomeModule,
+    MatTooltipModule,
+    BasicAutocompleteComponent,
+  ],
+})
+export class AdminEntityEditComponent implements OnInit {
```

Comment:
I'd call this component `AdminEntityGeneralSettingsComponent`

Reply:
Hi @sleidig , just to confirm, do we also need to rename adminEntityEditComponent to AdminEntityGeneralSettingsComponent, including the corresponding test files, HTML specs, and SCSS files?

Reply:
renamed this component